flags.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "gopkg.in/alecthomas/kingpin.v2"
+
+var (
```

Comment:
Nit: I'm not sure how I feel about this one. I think go tends to prefer larger files vs isolated? I think all the values defined here are referenced in `stackdriver_exporter.go` so might make sense to keep it colocated.

Reply:
Like it, and would be definetely less change.